stable/distributed-jmeter/templates/jmeter-master-deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-master
```

Comment:
Resoure names should use the fullname template.

Reply:
@pedrocesar-ti can you please apply the changes?